src/service/ItemMapper.ts

```diff
@@ -22,21 +22,26 @@ export interface CheckDetailsItem {
 export abstract class ItemMapper {
 
     getCheckDetailsItem = (itemDetails: { companyName: string, companyNumber: string, itemOptions: CertificateItemOptions, deliveryDetails: DeliveryDetails | undefined }): CheckDetailsItem => {
-        return {
-            serviceUrl: `/company/${itemDetails?.companyNumber}/orderable/certificates`,
-            serviceName: SERVICE_NAME_CERTIFICATES,
-            titleText: "Certificate ordered",
-            pageTitle: "Certificate ordered confirmation",
-            happensNext: "We'll prepare the certificate and aim to dispatch it within " + dispatchDays + " working days.",
-            orderDetailsTable: this.getOrdersDetailTable(itemDetails),
-            certificateDetailsTable: 1,
-            deliveryDetailsTable: this.getDeliveryDetailsTable(itemDetails)
-        }
+        const happensText = itemDetails.itemOptions?.deliveryTimescale === "same-day" ? "We'll prepare the certificate and orders received before 11am will be dispatched the same day. Orders received after 11am will be dispatched the next working day." :
```

Comment:
This `const` is a little hard to read and the name `happensText` isn't very explanatory. A potential for improvement would be to move the text options into separate constants and rename the variable. E.g: 

```suggestion
        const whatHappensNextText = itemDetails.itemOptions?.deliveryTimescale === "same-day" ? SAME_DAY_HAPPENS_NEXT_TEXT : DEFAULT_TEXT
```